DKGRituals/schema.graphql

```diff
@@ -29,3 +32,18 @@ type RitualCounter @entity {
   unsuccessful: Int!
   notEnded: Int!
 }
+
+type Transaction @entity {
+  id: ID!
+  txHash: Bytes!
+  timestamp: BigInt!
+  from: Bytes!
+  to: Bytes
+  description: String
+  rituals: [Ritual!] @derivedFrom(field: "transactions")
+}
+
+type User @entity {
```

Comment:
Any obection to renaming this to `Initiator` as opposed to `User`?

Reply:
None, addressed in [update user entity name to initiator](https://github.com/nucypher/taco-subgraphs/pull/10/commits/5daa8b77f8f3e3ccecd8c45d17011a2eb33532c6)